Credentialer/ViewController.swift

```diff
@@ -8,18 +8,105 @@
+        let keyboardSize: CGSize = value.CGRectValue().size
+        let contentInsets: UIEdgeInsets = UIEdgeInsetsMake(0.0, 0.0, keyboardSize.height, 0.0)
+        scrollView.contentInset = contentInsets
+        scrollView.scrollIndicatorInsets = contentInsets
+        
+        // If active text field is hidden by keyboard, scroll it so it's visible
+        // Your app might not need or want this behavior.
+        var aRect: CGRect = self.view.frame
+        aRect.size.height -= keyboardSize.height
+        let activeTextFieldRect: CGRect? = activeTextField?.frame
+        let activeTextFieldOrigin: CGPoint? = activeTextFieldRect?.origin
+        if (!CGRectContainsPoint(aRect, activeTextFieldOrigin!)) {
+            scrollView.scrollRectToVisible(activeTextFieldRect!, animated:true)
+        }
+    }
+    
+    func keyboardWillBeHidden(sender: NSNotification) {
+        let contentInsets: UIEdgeInsets = UIEdgeInsetsZero
+        scrollView.contentInset = contentInsets
+        scrollView.scrollIndicatorInsets = contentInsets
+    }
+    
+    func textFieldDidBeginEditing(textField: UITextField) {
+        activeTextField = textField
+        scrollView.scrollEnabled = true
+    }
+    
+    func textFieldDidEndEditing(textField: UITextField) {
+        activeTextField = nil
+        scrollView.scrollEnabled = false
+    }
+    
+    func getActiveTextField() {
+        usernameTextField.addTarget(self, action: Selector("textFieldDidBeginEditing:"), forControlEvents: UIControlEvents.EditingDidBegin)
+        usernameTextField.addTarget(self, action: Selector("textFieldDidEndEditing:"), forControlEvents: UIControlEvents.EditingDidEnd)
+        passwordTextField.addTarget(self, action: Selector("textFieldDidBeginEditing:"), forControlEvents: UIControlEvents.EditingDidBegin)
+        passwordTextField.addTarget(self, action: Selector("textFieldDidEndEditing:"), forControlEvents: UIControlEvents.EditingDidEnd)
+        cityTextField.addTarget(self, action: Selector("textFieldDidBeginEditing:"), forControlEvents: UIControlEvents.EditingDidBegin)
+        cityTextField.addTarget(self, action: Selector("textFieldDidEndEditing:"), forControlEvents: UIControlEvents.EditingDidEnd)
+        zipCodeTextField.addTarget(self, action: Selector("textFieldDidBeginEditing:"), forControlEvents: UIControlEvents.EditingDidBegin)
+        zipCodeTextField.addTarget(self, action: Selector("textFieldDidEndEditing:"), forControlEvents: UIControlEvents.EditingDidEnd)
+        countryTextField.addTarget(self, action: Selector("textFieldDidBeginEditing:"), forControlEvents: UIControlEvents.EditingDidBegin)
+        countryTextField.addTarget(self, action: Selector("textFieldDidEndEditing:"), forControlEvents: UIControlEvents.EditingDidEnd)
+    }
+    
+    func textFieldShouldReturn(textField: UITextField) -> Bool {
+        switch (textField){
+        case self.usernameTextField: self.passwordTextField.becomeFirstResponder()
```

Comment:
`case` statements should be split up in different lines...

```
case self.usernameTextField:
        self.passwordTextField.becomeFirstResponder()
```

increases readability